content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/zebra.rst

```diff
@@ -0,0 +1,221 @@
+
+In Odoo, labels printed in the Zebra Programming Language (ZPL) file format is designed to fit a
+four-by-six inch label. To resize or format text to fit a variety of |ZPL| label sizes,
+:ref:`navigate to the ZPL label view <inventory/shipping_receiving/zpl-view>` and alter the |ZPL|
+code.
+
+Refer to the following sections for explanations and example code for frequently requested Zebra
+label customizations.
+
+- :ref:`adjust margins <inventory/shipping_receiving/margin>`
+- :ref:`enlarge/minimize barcodes <inventory/shipping_receiving/resize>`
+- :ref:`rotate elements <inventory/shipping_receiving/rotate>`
+
+.. _inventory/shipping_receiving/zpl-view:
+
+Navigate to ZPL label view
+==========================
+
+To begin customizing a Zebra label in Odoo, turn on :ref:`developer mode <developer-mode>`, and on
+the Odoo's main dashboard, type in `Reports`. From the search results, choose
+:guilabel:`Settings/Technical/Reporting/Reports` to open the :guilabel:`Reports page`.
+
+.. note::
+   To manually navigate to the :guilabel:`Reports` page, go to :menuselection:`Settings app -->
+   Technical --> Reporting: Reports`.
+
+.. image:: zebra/search.png
+   :align: center
+   :alt: Show global search result for "Reports".
+
+On the :guilabel:`Reports` page, in the :guilabel:`Search...` bar, type `ZPL` to get a list of
+available Zebra labels in Odoo. Select the desired Zebra label to modify.
+
+.. figure:: zebra/qweb-views.png
+   :align: center
+   :alt: Show Qweb smart button on the Lot and Serial Number (ZPL) report.
+
+   **Lot and Serial Number (ZPL)** report, highlighting the Qweb smart button.
+
+.. note::
+   Printable ZPL labels in Odoo:
+
+   - finished product
+   - :ref:`lot/serial number <inventory/shipping_receiving/lot-sn-labels>`
+   - operation type
+   - package barcode
+   - :ref:`product label <inventory/shipping_receiving/product-labels>`
+   - product packaging
+
+Next, select the :icon:`fa-code` :guilabel:`Qweb Views` smart button, and choose the desired label
+:doc:`view <../../../../../developer/reference/user_interface/view_records>`. On the view page, go
+to the :guilabel:`Architecture` tab to view the |ZPL| code.
+
+.. important::
+   To ensure the customization are not overwritten during an update, click the :icon:`fa-bug`
+   :guilabel:`(bug)` icon on the view page. Then, select :guilabel:`View Metadata` option to open
+   the pop-up window. Then, ensure the :guilabel:`No Update` field is set to :guilabel:`true
+   (change)`.
+
+.. image:: zebra/architecture.png
+   :align: center
+   :alt: Architecture tab in the view.
+
+.. _inventory/shipping_receiving/margin:
+
+Adjust margin
+=============
+
+Text gets cut off from standard |ZPL| labels printed in Odoo when the line exceeds fifty-five
+characters. To fit long product names or lot numbers on a single line, adjust the margin.
+
+To begin, navigate to the :ref:`ZPL code of the label <inventory/shipping_receiving/zpl-view>`. In
```

Comment:
This is all happening in the 'Architecture' tab of the view form, correct? Might be worth considering a quick reminder about that, just in case the reader gets a bit lost. Totally optional, though. 👍